### ISSUE TITLE
Missing git tag 3.1

### ISSUE THREAD

**Issue by xhalo32** [plastex/plastex#386]
Hi,

I'm adding this python package to [nixpkgs](https://github.com/NixOS/nixpkgs/) and noticed that this GitHub repository seems to be missing the tag for 3.1 (which probably should point to `193747318f7ebadd19eaaa1e9996da42a31a2697`).

Cheers,
Niklas

**Comment by eric-wieser** [comment]
Agreed, https://github.com/plastex/plastex/commit/193747318f7ebadd19eaaa1e9996da42a31a2697 looks like the right commit to point to. @PatrickMassot, any chance you could create this tag?

**Comment by PatrickMassot** [comment]
Thanks for the ping. Somehow only the pings reach me those days :(  I pushed the requested tag.
